BaseStation/MapDownloadTest/Program.cs

```diff
@@ -3,6 +3,8 @@
 using System.IO;
 using System.Collections.Generic;
 using HuskyRobotics.Utilities;
+using System.Security.Cryptography;
+using System.Threading;
 
 // make sure there is a folder named MapTiles in the working directory
 // google API key: AIzaSyDr7Tuv6bar9jkYbz23b3jv0RlHLnhtzxU
```

Comment:
Be careful not to check in API keys. You might want to invalidate your old key and get a new one since this repo is public.